File: src/main/java/com/monis/guess/GameFor.java
package com.monis.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        /*int sum = 0;
        for (int i = 1; i <= 10; i++) {
            sum = sum +i;
        }
        System.out.println(sum);*/

        Scanner scanner = new Scanner(System.in);
        int secret = new Random().nextInt(10) + 1;
        System.out.println(secret);
        for (int i = 1; i <= 4; i++) {
            System.out.println("Please enter a number :("+i+"/4):");
            int number = scanner.nextInt();
            System.out.println("第" + i + "次" + number);
            if (number > secret) {
                System.out.println("higher");
            } else if (number < secret) {
                System.out.println("lower");
            } else {
                System.out.println("Great , the number is : " + number);
                break;
            }


            /*if (number == -1) {
                break;
            }*/
        }
        System.out.println("結束");
    }
}

File: src/main/kotlin/com/monis/GameFor.kt
package com.monis

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10) + 1
    println(secret)
    val scanner = Scanner(System.`in`)
    for (i in 1..4) {
        print("Please enter a number(${i}/4):")
        var number = scanner.nextInt()
        println("第${i}次 $number")
        if (number < secret) {
            println("higher")
        }else if (number > secret) {
            println("lower")
        } else {
            println("Great! the number is: " + number)
            break
        }
    }
}